Web/Case Study Assignment-5/CaseStudyAssignment-5/WebContent/addCar.jsp

```diff
@@ -0,0 +1,81 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+    
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+		<title>Car Portal</title>
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<link rel="stylesheet" type="text/css" href="css/styleLogin.css">
+	</head>
+	<body>
+		<header> 
```

Comment:
ACTION: Do we have any way to create a single view for header and use on other view?